django/tasks/constants.py

```diff
@@ -0,0 +1,4 @@
+UNKNOWN = 0
+PENDING = 1
+FAILED = 2
+SUCCESS = 3
```

Comment:
agreed with @apollo13 - we use this everywhere, either we change all or stay consistent. Either way not in scope of current PR